SteamList/Controllers/GameDetailsViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  GameDetailsViewController.swift
+//  SteamList
+//
+//  Created by Macbook on 03/01/2023.
+//
+
+import UIKit
+
+class GameDetailsViewController: UIViewController {
```

Comment:
sorry I forgot about init)
fixed!

Reply:
I removed initializers since now I am not injecting game object by constructor rather I am modifying its value before navigation takes place to game details page